geth/jail/jail_test.go

```diff
@@ -115,6 +118,52 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+
+	defer func() {
+		if err := recover(); err != nil {
+			require.NotNil(err, "Panic occured")
+		}
+	}()
+
+	require.NotNil(s.jail)
+
+	s.StartTestNode(params.RopstenNetworkID)
+	defer s.StopTestNode()
+
+	// load Status JS and add test command to it
+	s.jail.BaseJS(baseStatusJSCode)
+	s.jail.Parse(testChatID, rxSendJS)
+
+	cell, err := s.jail.GetCell(testChatID)
+	require.NoError(err)
+	require.NotNil(cell)
+
+	var wg sync.WaitGroup
+
+	// internally (since we replaced `web3.send` with `jail.Send`)
+	// all requests to web3 are forwarded to `jail.Send`
+	for i := 0; i < 5; i++ {
+		go func() {
+			wg.Add(1)
+			defer wg.Done()
+
+			_, err = cell.Run(`
+				_status_catalog.commands.sendAsync({
```

Comment:
String should be a constant outside the block, like in `rxsend_test.go`, to reduce nesting.